src/go/k8s/cli/configurator/main.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// main package. This line should be remove after merging the following
+// https://github.com/vectorizedio/redpanda/pull/753
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/afero"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"gopkg.in/yaml.v3"
+)
+
+var (
+	hostName		string
+	svcFQDN			string
+	configSourceDir		string
+	configDestination	string
+	redpandaRPCPort		int
+)
+
+var errorMissingEnvironmentVariable = errors.New("missing environment variable")
+
+func main() {
+	if err := checkEnvVars(); err != nil {
```

Comment:
it would be nicer if this method returns struct with all those field rather than mutate some global state in this file

Reply:
Ok will do